packages/server/src/utilities/redis.ts

```diff
@@ -25,9 +24,7 @@ export async function init() {
 
   // Duplicate the socket client for pub/sub
   socketClient = await redis.clients.getSocketClient()
-  if (!env.isTest()) {
-    socketSubClient = socketClient.getClient().duplicate()
-  }
```

Comment:
This was only gated because we used `ioredis-mock`. Now that's gone we can use Redis proper.